packages/x-tree-view/src/TreeItem2DragAndDropOverlay/TreeItem2DragAndDropOverlay.tsx

```diff
@@ -24,26 +24,26 @@ const TreeItem2DragAndDropOverlayRoot = styled('div', {
       style: {
         marginLeft: 'calc(var(--TreeView-indentMultiplier) * var(--TreeView-itemDepth))',
         borderRadius: theme.shape.borderRadius,
-        backgroundColor: alpha((theme.vars || theme).palette.primary.dark, 0.15),
+        backgroundColor: alpha(theme.palette.primary.dark, 0.15),
```

Comment:
The correct solution can be found here: https://github.com/mui/mui-x/blob/3f12d3de1b6961105493978f48c532375289f7fa/packages/x-tree-view/src/TreeItem2/TreeItem2.tsx#L84-L86
Would you be able to apply the necessary changes? 🤔

Reply:
ok